src/routes/config.ts

```diff
@@ -0,0 +1,45 @@
+import * as _ from 'lodash';
+import { RequestHandler } from 'express';
+import { JSONSchema6 } from 'json-schema';
+
+import {
+	RESERVED_NAMES,
+	RESERVED_NAMESPACES,
+	INVALID_CHARACTER_REGEX,
+	WHITELISTED_NAMES,
+	WHITELISTED_NAMESPACES,
+	BLACKLISTED_NAMES,
+    SUPERVISOR_CONFIG_VAR_PROPERTIES,
```

Comment:
Weird spacing here. `npm run prettify` should be available on current master (otherwise it'll be introduced by #1).